scs-app/src/app/fulfill_major/fulfill_major.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { debounceTime, distinctUntilChanged, map } from 'rxjs/operators';
+
+import { CreditCalcService } from '../services/credit-calc/credit-calc.service';
+
+@Component({
+  selector: 'fulfill-major',
+  templateUrl: './fulfill_major.component.html',
+  styleUrls: ['./fulfill_major.component.scss'],
+  providers: [ CreditCalcService ]
+})
+export class FulfillMajorComponent implements OnInit {
+
+  private courselist: {course_name: string, fulfills: string}[] = [];
+  public user_courses: {course_name: string, fulfills: string}[] = [];
```

Comment:
so this is what the user selects I'm assuming? let's put this all in the same camel notation